src/kubernetes_cluster/proof/controller_runtime_liveness.rs

```diff
@@ -286,188 +287,49 @@ pub proof fn lemma_from_some_state_to_arbitrary_next_state_to_reconcile_idle<K:
     );
     leads_to_trans_n!(
         spec,
-        lift_state(at_reconcile_state(cr.object_ref(), state)),
+        lift_state(at_expected_reconcile_states(cr.object_ref(), state)),
         lift_state(at_some_state_and_pending_req_in_flight_or_resp_in_flight),
         lift_state(at_expected_reconcile_states(cr.object_ref(), next_state)),
         lift_state(|s: State<K, R>| !s.reconcile_state_contains(cr.object_ref()))
     );
 }
 
-#[verifier(external_body)]
```

Comment:
So these lemmas are no longer useful?

Reply:
The origin ones may not be used any more. But later some similar ones may be added.